oasisriddle/docs/riddle_eng

```diff
@@ -0,0 +1,62 @@
+# Oasis Riddle
+
+Developers at [Oasis Network][oasis-network] have created a short riddle
+that unlocks the path to a ROSE treasure. They hid it on the [Oasis Sapphire][oasis-explorer]
+blockchain and your job now is to write a script connecting to the
+smart contracts already deployed on Sapphire that will enable you to obtain
+the secret questions. If you manage to answer all the questions correctly, 
+you'll be able to collect your ROSE treasure! 🌹
+
+## What if I don't exactly know how Blockchains, Ethereum, Solidity, Dapps work?
```

Comment:
This title was for a national programming competition. For Consensus, I suggest something else:

```suggestion
## I like blockchains, but I'm not a skilled programmer and  I haven't really written any smart contract yet
```